girder/taskflow/server/models/taskflow.py

```diff
@@ -61,6 +61,7 @@ def append_to_log(self, taskflow, log):
             }
         }
 
+        send_log_notification('taskflow', taskflow, log)
```

Comment:
I wasn't entirely sure if this is the right place to send the log notification from.